lua/autorun/sh_portal_movement.lua

```diff
@@ -35,12 +35,26 @@ local function updateCalcViews(finalPos, finalVel)
 			finalPos = ply:EyePos()
 			SeamlessPortals.DrawPlayerInView = false
 		end
+		
+		local wep = ply:GetActiveWeapon()
+		if wep:IsValid() then
+			if isfunction(wep.CalcView) then
```

Comment:
Was just looking through for fun when I noticed that you have a redundant double if statement while the other hook uses fixes this issue...